catalog/transformer/catalog-transformer-mgmp/src/main/java/org/codice/alliance/catalog/converter/mgmp/MgmpConverter.java

```diff
@@ -0,0 +1,874 @@
+                "mediaReferenceNo");
+        pathValueTracker.add(new Path(MgmpConstants.ASSOCIATIONS_RELATED_TYPE_CODE_LIST_PATH),
+                MgmpConstants.MGMP_ASSOCIATION_TYPE_CODE);
+        pathValueTracker.add(new Path(MgmpConstants.ASSOCIATIONS_RELATED_TYPE_CODE_LIST_VALUE_PATH),
+                "mediaAssociation");
+        pathValueTracker.add(new Path(MgmpConstants.ASSOCIATIONS_RELATED_TYPE_TEXT_PATH),
+                "Media Association");
+    }
+
+    private void addMdIdentificationTopicCategories() {
+        addMultiValues(getValues(Topic.CATEGORY), MgmpConstants.GMD_TOPIC_CATEGORY_PATH);
+    }
+
+    private void addMdIdentificationDescriptiveKeywords() {
+        addMultiValues(getValues(Topic.KEYWORD), MgmpConstants.GMD_KEYWORD_PATH);
+    }
+
+    private void addGmdMetacardFormat() {
+        addFieldIfString(Media.FORMAT, GmdConstants.FORMAT_PATH);
+        addFieldIfString(Media.FORMAT_VERSION, GmdConstants.FORMAT_VERSION_PATH);
+    }
+
+    private void addMdIdentificationAbstract() {
+        addFieldIfString(Core.DESCRIPTION, GmdConstants.ABSTRACT_PATH);
+    }
+
+    private void addGmdTitle() {
+        addFieldIfString(() -> {
+            return Optional.of(StringUtils.defaultString(metacard.getTitle()));
+        }, GmdConstants.TITLE_PATH);
+    }
+
+    private void addFileIdentifier() {
+        addFieldIfString(() -> {
+            return Optional.of(StringUtils.defaultString(metacard.getId()))
+                    .map(this::formatId);
+        }, GmdConstants.FILE_IDENTIFIER_PATH);
+    }
+
+    private void addHierarchyLevel() {
+        pathValueTracker.add(new Path(GmdConstants.CODE_LIST_VALUE_PATH), "dataset");
+        pathValueTracker.add(new Path(GmdConstants.CODE_LIST_PATH), MgmpConstants.MGMP_SCOPE_CODE);
+    }
+
+    @Override
+    protected String getRootNodeName() {
+        return GmdConstants.GMD_LOCAL_NAME;
+    }
+
+    private void addMetadataReleasability() {
+        Attribute releasibilityAttribute = metacard.getAttribute(Security.METADATA_RELEASABILITY);
+        Attribute disseminationAttribute = metacard.getAttribute(Security.METADATA_DISSEMINATION);
+
+        if (isReleasabilityAndDisseminationSet(releasibilityAttribute, disseminationAttribute)) {
+
+            String value = disseminationAttribute.getValue()
+                    .toString() + " " + join(releasibilityAttribute.getValues(), "/");
+
+            pathValueTracker.add(new Path(MgmpConstants.METADATA_RELEASABILITY_PATH), value);
+
+        }
+
+    }
+
+    private boolean isReleasabilityAndDisseminationSet(Attribute releasibilityAttribute,
+            Attribute disseminationAttribute) {
+        return releasibilityAttribute != null && disseminationAttribute != null
+                && releasibilityAttribute.getValues() != null && !releasibilityAttribute.getValues()
+                .isEmpty() &&
+                disseminationAttribute.getValue() != null
```

Comment:
Don't need to check for null since `isNotBlank()` will return false if the argument is null.